packages/kit/src/runtime/client/renderer.js

```diff
@@ -392,6 +392,7 @@ export class Renderer {
 
 		/** @type {import('./types').NavigationResult} */
 		const result = {
+			redirect: branch.find((b) => b?.loaded?.redirect)?.loaded?.redirect,
```

Comment:
We can't use optional chainning unfortunately because we don't transpile the code and Node 14 does not support it in all versions.

Reply:
Thank you for reviewing.

OK. Should I use Node12 for kit?

Reply:
You don't need to do that, you just have to remember this caveat - and the CI will catch this, too, in case it's overlooked.

Reply:
Thank you for telling me!
And I updated it.